src/VTK.jl

```diff
@@ -1,40 +1,39 @@
-function get_cell_type(nen, ndim)
-    # TODO: This is a bad way of figuring out the eltype
+# Deprecated function
+function vtk_grid{T}(topology::Matrix{Int}, coord::Matrix{T}, filename::AbstractString)
```

Comment:
Makes sense. It is quite nice to not have to specify but if we get a proper mesh structure then it would know. Can remove for now I guess.